### ISSUE TITLE
[Feature Request] Border colors read from preferences rather than hardcoded

### ISSUE THREAD

**Issue by blahsd** [w0lfschild/winBuddy#28]
This would allow to change border color in according to the color scheme in use. Currently it's basically impossible to automate because it would require recompiling every time.

**Comment by w0lfschild** [comment]
You mean like light or dark based on the system setting?

**Comment by blahsd** [comment]
Nope. I meant it as in – I change frequently the colorscheme of my terminal, wm, etc. according to fixed colorschemes (through pywal). So each time I set a system-wide colorscheme, I'm left with some 16 colors that all belong to the same palette. I would like to be able to write this colors down somewhere – be it a config file, or straight to defaults – so that winBuddy would read it and color the borders accordingly, at least after a restart. As an example:

I set up my color palette to be orange, green, yellow, red. I'd like winBuddy to read my preferences and set active borders orange and non-active green. 

I then change my color palette to be blue, purple, red, pink. I'd like winBuddy to read the preferences set active borders blue and non-active green. 

I'm almost positive it can be done – unfortunately I might be talking out of my ass because I know no obj-c. What I'm thinking is something along the way of:

`    [borderWin.contentView setBorderColor:self.isKeyWindow ? [ReadPref(@"AtiveBorderColor") NSColor] : [[ReadPref(@"NonAcriveBorderColor") NSColor]]];`

Then you just need to write in the preferences what colors you want to use. 

I hope it made sense, idk because I have some experience developing but none with obj-c and in general Xcode really eludes me.
